doc/development/documention.rst

```diff
@@ -31,12 +31,12 @@ like this::
   xdg-open _build/html/index.html
 
 
-Please watch out for any warning and error while building the documentation.
-In your browser check if the markup is
+Please watch out for any warnings and errors while building the documentation.
+In your browser check if the markup is valid prior to pushing your changes and creating the pull-request please run::
 
-Prior to pushing your changes and creating the pull-request please run::
-
-  make clean html
+  make clean
+  ...
+  make html
```

Comment:
I assume this is what you refer to as the "invalid commend". I'm just curious, why this should be invalid.

Reply:
`make clean html` just cleans and ignores the second argument on windows. Not sure on Linux, but best to be safe for both.

Reply:
It's because on windows, `make.bat` is run, not `Makefile`. They're different scripts.